### PR TITLE
Implement basic data generator

### PR DESCRIPTION
**Pull request by hammertube**

### CODE REVIEW

File: client/src/components/People.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+class People extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+    };
+  }
```
Comment (dorktobes) [inline client/src/components/People.jsx]: This should be a functional-stateless component.

File: client/src/components/Time.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <select>
+        <option value="2">2 people</option>
+        <option value="3">3 people</option>
+        <option value="4">4 people</option>
+        <option value="5">5 people</option>
+        <option value="6">6 people</option>
+        <option value="7">7 people</option>
+        <option value="8">8 people</option>
+        <option value="9">9 people</option>
+        <option value="10">10 people</option>
```
Comment (dorktobes) [inline client/src/components/Time.jsx]: Looks like this was borrowed from the `People` component.

File: client/src/index.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  render() {
+    return (
+      <div className="island-item">
+        <h3>Make a Reservation</h3>
+        <form className="reservation-availability-search-form">
+          <ul>
+            <li><Calendar /></li>
+            <li><Time /></li>
+            <li><People /></li>
+          </ul>
```
Comment (dorktobes) [inline client/src/index.jsx]: Is there a compelling reason that this is a list?

File: database/index.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
+const Sequelize = require('sequelize');
+
+const sequelize = new Sequelize('yelp', 'root', '', {
```
Comment (dorktobes) [inline database/index.js]: Maybe don't name your db after the original site.

File: package.json [hunk 5]
```diff
@@ -9,12 +9,11 @@
   },
   "scripts": {
     "react-dev": "webpack -d --watch",
-    "bulbasaur": "node database/initializer.js",
+    "bulbasaur": "node database/index.js",
```
Comment (dorktobes) [inline package.json]: This is a poorly named script. Variable naming is important in all your files, not just the js.

File: database/schema.sql [hunk 6]
```diff
@@ -26,7 +26,7 @@ CREATE TABLE reservations (
   month int NOT NULL,
   year int NOT NULL,
   time int NOT NULL,
-  numofpeopel int NOT NULL,
+  numofpeople int NOT NULL,
   restaurants_id int NOT NULL,
```
Comment (dorktobes) [inline database/schema.sql]: Aren't you defining your schema in `database/index.js`? Is this file still necessary?

File: server/index.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+const bodyParser = require('body-parser');
+
+const cors = require('cors');
+
+const port = 6666;
+
+const app = express();
+
+app.use(express.static('./client/dist'));
```
Comment (dorktobes) [inline server/index.js]: You might want to use the `path` module here as well.

File: .vscode/settings.json [hunk 8]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
[the remark this answers is not in the document]
Comment (hammertube) [reply .vscode/settings.json]: Sure! I am adding it to .gitignore now.

File: database/schema.sql (hunk 6 above)
Comment (hammertube) [reply database/schema.sql]: I forgot to delete this file 👀

File: client/src/components/Time.jsx (hunk 2 above)
Comment (hammertube) [reply client/src/components/Time.jsx]: I'll change it in my second PR.
